grid_map_msgs/srv/ClearSubGridMap.srv

```diff
@@ -0,0 +1,4 @@
+# Sub map
```

Comment:
I think there's some confusion about the term "submap". So far I used it to describe a map which lies inside the original map but is smaller. In your case, the maps are of the same size, so submap does not apply.

My suggestions is to not use submap, but call rename the service to "ProcessMap.srv". The topic in Elevation Mapping can then be more specific.